db/migrate/20170810164131_rename_provisioning_keys_to_environments.rb

```diff
@@ -0,0 +1,22 @@
+class RenameProvisioningKeysToEnvironments < ActiveRecord::Migration[5.0]
+  def change
+    rename_table :provisioning_keys, :environments
+    rename_column :environments, :api_env, :name
+    rename_column :environments, :content, :provisioning_key
+
+    add_reference :access_requests, :environment
+    add_reference :tokens,          :environment
+
+    Token.all.each do |token|
+      env = Environment.find_by(name: token.api_env)
+      token.environment_id = env.id
+      token.save!
+    end
+
+    AccessRequest.all.each do |access_request|
+      env = Environment.find_by(name: access_request.api_env)
+      access_request.environment_id = env.id
+      access_request.save!
+    end
```

Comment:
Ah yes, I had intended to do this after testing the migration but forgot.  Thanks for pointing it out.